getting-started-dao.adoc

```diff
@@ -0,0 +1,214 @@
+
+Don't forget this step! If your vote reveal transaction is not published, your vote won't be decrypted, and it won't be counted.
+
+=== 4. Get vote results
+
+Once the vote reveal phase is over, results are published and the next voting cycle starts. You can see results on the `Vote results` section of the `Governance` tab of the `DAO` screen.
+
+.See voting results.
+image::dao-vote-results.png[See voting results]
+
+As you can see, our proposal was accepted. Hopefully you have similar luck!
+
+If your proposal was a compensation request, this is when the BTC you included in your request will show as BSQ (after having been 'colored' by the voting process). Check your BSQ wallet balance in the `BSQ Wallet` tab of the `DAO` screen to see the updated balance.
+
+That's it—you've now participated in a full voting cycle of the Bisq DAO, the upcoming decentralized governance mechanism for the Bisq exchange. Bravo!
+
+== Explore a BSQ block explorer
+
+Since BSQ is just bitcoin, you can see BSQ transactions on any bitcoin block explorer. But because these bitcoins are colored, ordinary bitcoin block explorers can't recognize which bitcoin transactions are also BSQ transactions.
+
+So there's a https://explorer.bisq.network/testnet/[BSQ block explorer^]. Whenever you do a BSQ transaction, you can see it there.
+
+See the difference: here's https://explorer.bisq.network/tx.html?tx=3dc364b55fa9e92058d52a9c09b98300781414573f123e63367ff3749a8615b2[a simple transaction^] on the BSQ block explorer that shows the transfer of 500 BSQ from one wallet to another. https://blockstream.info/testnet/tx/3dc364b55fa9e92058d52a9c09b98300781414573f123e63367ff3749a8615b2/[Here's that same transaction^] on an ordinary bitcoin block explorer.
+
+Here's a screenshot of the past few BSQ transactions as of mid-December 2018. You'll notice that this block explorer recognizes the various transaction properties that make valid BSQ transactions, and it labels them accordingly:
+
+.Some BSQ transactions on the https://explorer.bisq.network[BSQ block explorer^].
+image::bsq-block-explorer.png[Some recent BSQ transactions]
+
+== Next steps
+
+At the moment, the Bisq DAO is being tested thoroughly. If you'd like to help, there are bounties! https://bisq.community/t/how-to-explore-the-dao-on-testnet/6692[See more details here^].
+
+Otherwise, check out our <<user-dao-intro#,conceptual overview of the Bisq DAO>> and our https://www.youtube.com/playlist?list=PLFH5SztL5cYOLdYJj3nQ6-DekbjMTVhCS[video series^] on DAO concepts.
+
+== Get help and stay in touch
+
+If you get stuck, reach out! There's a community of people to help you on the https://bisq.community/[Bisq forum^], the https://www.reddit.com/r/bisq/[/r/bisq subreddit^], and the https://bisq.network/slack-invite[Bisq Slack team^].
+
```

Comment:
Maybe sort by priority (e.g. reddit at the end - or remove it as we dont monitor reddit closely)

Reply:
I kept reddit (but put it last) since it seems that Alexej checks and responds regularly.